source/common/upstream/load_balancer_impl.h

```diff
@@ -436,6 +428,14 @@ class ZoneAwareLoadBalancerBase : public LoadBalancerBase {
   Common::CallbackHandlePtr priority_update_cb_;
   Common::CallbackHandlePtr local_priority_set_member_update_cb_handle_;
 
+  // Config for zone aware routing.
+  const uint64_t min_cluster_size_;
+  const uint32_t routing_enabled_;
+  const bool fail_traffic_on_panic_ : 1;
```

Comment:
nit: please add a small note about the coalescing of these data members at the end (something like [this](https://github.com/envoyproxy/envoy/blob/1ac0a5948ca317fd418777dab2ffe27935c0a276/source/common/router/config_impl.h#L342)).

Reply:
Done